src/components/AccountCard.vue

```diff
@@ -29,14 +29,16 @@ function focusInput(event: MouseEvent) {
 
 <template>
     <div class="account-card">
-        <a class="admin-switch" href="javascript:void(0)" @click="account.admin = !account.admin" title="Changer le rôle">
+        <a class="admin-switch" href="javascript:void(0)" @click="dataStore.editAccount(id, 'admin', !account.admin)"
+            title="Changer le rôle">
```

Comment:
For clarity create function in <script> invoked in template, and then call store in them

i.e

```
@click="editAccount"

<script setup lang="ts">

const editAccount = () => {
  dataStore.editAccount(id, 'admin', !account.admin)"
}

</script>
```

(or set parameter for the editAccount function, if somes are coming from the template (like in V-for..etc)